src/nni_manager/training_service/remote_machine/remoteMachineTrainingService.ts

```diff
@@ -359,7 +351,7 @@ class RemoteMachineTrainingService implements TrainingService {
      * Get culster metadata
      * @param key metadata key
      */
-    public getClusterMetadata(key: string): Promise<string> {
+    public getClusterMetadata(): Promise<string> {
```

Comment:
why remove the key?

Reply:
it causes a TSlint warning that not used value `key`, so remove it.

Reply:
This function is declared in `abstract class TrainingService`, every trainingService will implement `public abstract getClusterMetadata(key: string): Promise<string>;` , the key should not be removed.

Reply:
fixed by change the argument name to _key.